app/src/main/kotlin/com/wire/android/ui/home/HomeScreen.kt

```diff
@@ -44,10 +44,10 @@ fun HomeScreen(startScreen: String?, viewModel: HomeViewModel, syncViewModel: Sy
     val homeState = syncViewModel.homeState
     val snackbarHostState = remember { SnackbarHostState() }
 
-    handleSnackBarMessage(snackbarHostState, viewModel.snackbarMessageState, viewModel::clearSnackbarMessage)
+    handleSnackBarMessage(snackbarHostState, homeUIState.snackbarState, homeUIState::clearSnackbarMessage)
 
     LaunchedEffect(viewModel.savedStateHandle) {
-        viewModel.checkPendingActions()
+        viewModel.checkPendingActions()?.let(homeUIState::setSnackBarState)
```

Comment:
plx change checkPendingActions to something related with snackbar as it will handle the snackbar anyway